datalad_osf/remote.py

```diff
@@ -23,29 +23,49 @@ class OSFRemote(SpecialRemote):
        Write doc section on how to do that, and what URL should be used to
        configure the special remote.
 
+
     Initialize the special remote::
 
-       git annex initremote osf type=external externaltype=osf \\
+       git annex initremote dolphinarchive type=external externaltype=osf \\
             encryption=none project=https://osf.io/<your-component-id>/
 
-    However, you may reuse an existing project without overwhelming it with
-    garbled filenames by setting a path where git-annex will store its data::
+    Authenticate (optionally):
 
-       git annex initremote osf type=external externaltype=osf \\
-            encryption=none project=https://osf.io/<your-component-id>/ \\
-            objpath=git-annex
+    The special remote is usable in a read-only form when configured like this,
+    which is perfectly good for most scientific applications.  But to upload a
```

Comment:
This statement assumes public projects, correct? In my personal usage of OSF in three consortiums, everything was done in private projects. I'd nevertheless consider that work "scientific" ;-)

Reply:
Okay, maybe it was kind of snarky. I wanted to emphasize why you would care about the difference between anonymous and credentialed auth. Maybe @SRSteinkamp and the other half of the team can help me convey that?